packages/manager/modules/billing/src/billing.scss

```diff
@@ -0,0 +1,5 @@
+@import 'bootstrap4/scss/_functions.scss';
```

Comment:
Not scoped to billing module ? we might encounter issues if imported in Dedicated with bootstrap3